components/AppBar/docs/tracking-a-scroll-view.md

```diff
@@ -1,5 +0,0 @@
-### Tracking a scroll view
```

Comment:
Should we keep this around if people are still using the old API?

Reply:
This document was a duplicate of the flexible header docs, which I'm now linking to.